themes/default/content/blog/pulumi-release-notes-80/index.md

```diff
@@ -0,0 +1,281 @@
+
+- [Pulumi Deployments](/blog/pulumi-deployments): The Pulumi Service now supports executing infrastructure deployments as a managed service, driven by click-to-deploy in the Console, Git push-to-deploy from GitHub, programmatic deployments via the Pulumi Service REST API as well as automation via the Pulumi Automation API. [Pulumi Deployments](/blog/pulumi-deployments) enables organizations to scale up their infrastructure automation. Learn more in the [Pulumi Service section](#pulumi-service--pulumicom) below!
+- [Pulumi YAML GA](/blog/pulumi-yaml-ga): In addition to supporting rich general purpose programming languages like TypeScript, Python, Go, .NET and Java, Pulumi added support last May for Pulumi YAML as a simple, declarative interface for accessing the full breadth of the Pulumi Universal Infrastructure as Code platform. Pulumi YAML is now Generally Available as part of its 1.0 release, enabling everyone in an organization to use Pulumi to manage cloud infrastructure.
+- [Architecture Templates](/blog/intro-architecture-templates): New templates providing architecture-specific starting points for the most common infrastructure as code use cases - Containerized Services, Kubernetes Clusters, Serverless Applications, Static Websites and Virtual Machines - available across all major cloud platforms and supported languages.
+- [Project Level Configuration](/blog/project-config-mvp): As cloud infrastructure projects grow in complexity, Pulumi provides many tools to manage the complexity of the software that defines these projects. But the configuration of that software also grows in complexity. New support for Project-Level Configuration across all Pulumi languages makes it easier to manage complex configuration as cloud projects scale up.
+
+👉  Learn more in the [Cloud Engineering Days Announcements blog](/blog/nov-2022-launches).
+
+## Pulumi CLI and core technologies
+
+### Pulumi CLI now displays time elapsed per resource
+
+When you modify a resource the Pulumi CLI will display the time elapsed so you can determine how long the modification elapsed for. This change helps surface for each resource the time it takes to create or modify allowing you to plan for it next time. See it in action below!
+
+[![Watch it in action](https://asciinema.org/a/XHYWenULkugoY57kStsadcBH8.svg)](https://asciinema.org/a/XHYWenULkugoY57kStsadcBH8)
+
+👉  Learn more in the [Elapsed Time per Resource blog post](blog/pulumi-cli-time-elapsed) and in the [resource creation times GitHub issue](https://github.com/pulumi/pulumi/issues/10534).
+
+### Automation API NodeJS parallel inline programs
+
+Automation API now supports parallel execution of NodeJS inline programs in addition to the existing Python inline program support. Now users can run multiple NodeJS Pulumi programs in parallel when they are using Automation API. It uses NodeJS AsyncLocalContext to isolate state away from process environments and into context-specific stores owned by LocalWorkspaces.
```

Comment:
You can also mention .NET as a supported runtime for parallel execution (it has worked since a long time ago)